rules/aip0142/time_field_names_test.go

```diff
@@ -31,6 +31,8 @@ func TestFieldName(t *testing.T) {
 		{"InvalidIsMistake", "google.protobuf.Timestamp", "created", testutils.Problems{{Suggestion: "create_time"}}},
 		{"InvalidContainsMistake", "google.protobuf.Timestamp", "last_modified", testutils.Problems{{Suggestion: "update_time"}}},
 		{"InvalidNoSuffix", "google.protobuf.Timestamp", "create", testutils.Problems{{Message: "should end"}}},
+		{"InvalidNoSuffixRepeatedPlural", "repeated google.protobuf.Timestamp", "sample_times", testutils.Problems{}},
+		{"InvalidNoSuffixRepeated", "repeated google.protobuf.Timestamp", "sample_time", testutils.Problems{}},
```

Comment:
These look like `Valid` test cases (there are no `Problems` emitted), but the test case is named `Invalid`. Should these have invalid test data that trigger a `Problem` or should they be green path test cases with a `Valid` name?

Reply:
Whoops. These are valid cases. Fixed the names and added an invalid test case.

Reply:
Thanks looks great.